cmd/info.go

```diff
@@ -79,10 +88,11 @@ func info(ctx *cli.Context) error {
 			continue
 		}
 
-		wb := utils.NewBuffer(8 + 8)
+		wb := utils.NewBuffer(8 + 9)
```

Comment:
For mountpoint using older version, what will happen?

Reply:
The last byte will be ignored, as older clients will always return the recursive result